src/style.css

```diff
@@ -1 +1,45 @@
 /* add styles here */
+html{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
There's a checklist item that suggests not to use tag names for styling, except for `html` and `body`.